src/cluster.ts

```diff
@@ -0,0 +1,51 @@
+import os from 'os';
+import cluster from 'cluster';
+import 'dotenv/config';
+import server from './server';
+import createLoadBalancer from './cluster/createMainServer';
+// import { PaDB } from './types';
+
+const PORT = process.env.PORT ? parseInt(process.env.PORT) : 4000;
+const workers: Array<any> = [];
+const cpuCount = os.cpus().length;
```

Comment:
accordingly to the task, available parallelism should be used:
```suggestion
const cpuCount = os.availableParallelism() - 1;
```